massgen/frontend/displays/textual_widgets/quickstart_wizard.py

```
@@ -444,15 +448,20 @@ def get_value(self) -> dict[str, str]:
             if "model" in value and self._model_select:
                 self._current_model = value["model"]
                 self._model_select.value = value["model"]
-            if "reasoning_effort" in value:
-                self._current_reasoning_effort = value["reasoning_effort"]
             self._update_reasoning_input()
+            if saved_reasoning_effort and self._reasoning_select:
+                try:
+                    self._reasoning_select.value = saved_reasoning_effort
+                    self._current_reasoning_effort = saved_reasoning_effort
+                except Exception:
+                    pass
```

Ruff (0.15.2)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)